CHANGELOG.rst

```diff
@@ -10,8 +10,17 @@ This document describes changes between each past release.
 
 - Remove pypy supports. (#1049)
 
+**Internal changes**
+
+- Permission schema children fields are now set during initialization instead of on
+  deserialization (#1046).
+- Request schemas (including validation and deserialization) are now isolated by method
+  and endpoint type (#1047).
+- Move generic API schemas (e.g TimeStamps and HeaderFields) from `kinto.core.resource.schema`
+  to a sepate file on `kinto.core.schema`.
```

Comment:
About the documentation: I couldn't fond mentions to these schemas anywhere. Also [in some parts](http://kinto.readthedocs.io/en/stable/core/resource.html?highlight=kinto.core.resource.schema#full-example) we use a colander String with url validator to implement an url instead of the URL schema.